lib/utils/docker-js.js

```diff
@@ -20,92 +20,6 @@
 import * as _ from 'lodash';
 import { ExpectedError } from '../errors';
 
-// Use this function to seed an action's list of capitano options
-// with the docker options. Using this interface means that
-// all functions using docker will expose the same interface
-//
-// NOTE: Care MUST be taken when using the function, so as to
-// not redefine/override options already provided.
-export const appendConnectionOptions = (opts) =>
-	opts.concat([
```

Comment:
I gather this code is no longer used after the conversion of the `build` and `deploy` commands to oclif.